dashboard/utils.py

```diff
@@ -61,6 +70,15 @@ async def run(self, server):
         :param server: Asyncio GRPC server.
         """
 
+    @staticmethod
+    @abc.abstractclassmethod
+    def is_optional():
```

Comment:
Prefer to have this static method for better maintainability. Alternative way is to just not load modules if it fails to import, but I think this way is more explicit and easy to figure out what modules are loaded with minimal installation.